pages/landing/TryBtn.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+const Trybtn = () => {
```

Comment:
O mesmo nome do arquivo deve se seguir para o componente, o nome do componente tá correto, usou pascalCase, só é necessário realizar o ajuste aqui

```suggestion
const TryBtn = () => {
```